projet/projects/tests/test_contribution.py

```diff
@@ -18,52 +18,53 @@ def setUp(self):
         self.second_project = Project.objects.create(
             user=self.second_user, title="Second one"
         )
-        self.contribution = Contributor.objects.create(
+        self.contributor = Contributor.objects.create(
             user=self.third_user, project=self.project
         )
         self.project_list_create = reverse("project-create")
         self.users_projects = reverse("users-projects")
         self.project_detail = reverse("update-delete", args=[self.project.id])
-        self.accept_contributor = reverse(
-            "accept-contribution", args=[self.project.id, self.contribution.id]
-        )
+        # self.accept_contributor = reverse(
+        #     "accept-contribution", args=[self.project.id, self.contribution.id]
+        # )
         self.project_contributions = reverse("contributors", args=[self.project.id])
         self.project_contributions_status = reverse(
             "pending-accepted-contributors", args=[self.project.id, "accepted"]
         )
+        self.invited = User.objects.create_user("invite@email.com", "somethingwoth")
 
     def test_owner_can_be_contributor(self):
         self.client.force_login(user=self.second_user)
-        contribute_url = reverse("contribute", args=[self.second_project.id])
-        response = self.client.post(contribute_url, format="json")
+        contribute_url = reverse("invite")
+        data = {"id": f"{self.second_project.id}", "email": f"{self.second_user.email}"}
 
-        self.assertEqual(response.status_code, 403)
+        response = self.client.post(contribute_url, data=data, format="json")
 
-    def test_contributor_can_offer_twice(self):
```

Comment:
Since users no longer offer to contribute to projects this test became obsolete and produced errors, so i removed it and created a new test that well fits the use case of sending invites.